### PR TITLE
Fix up code based on codespell

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
@edsantiago @mheon @TomSweeneyRedHat @ashley-cui @QiWang19 @nalind @saschagrunert PTAL

**Comment by nalind** [comment]
LGTM

**Comment by edsantiago** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
One reversal for consideration, otherwise
LGTM

**Comment by QiWang19** [comment]
LGTM

File: docs/tutorials/05-openshift-rootless-bud.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ drwxr-xr-x.  12 root   root    144 Jul  7 09:07 usr
 drwxr-xr-x.  18 root   root    235 Jul  7 09:07 var
 STEP 4: RUN dnf update -y | tee /output/update-output.txt
 Fedora 33 openh264 (From Cisco) - x86_64        817  B/s | 5.1 kB     00:06
-Fedora - Modular Rawhide - Developmental packag 3.0 MB/s | 3.1 MB     00:01
+Fedora - Modular Rawhide - Developmental package 3.0 MB/s | 3.1 MB     00:01
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply docs/tutorials/05-openshift-rootless-bud.md:323]: That is why I changed it, I don't think anyone will notice, and I don't want to deal with it in the future.  Although it does throw off the output.

File: docs/tutorials/05-openshift-rootless-bud.md [hunk 2]
```diff
@@ -415,4 +415,4 @@ registry.fedoraproject.org/fedora   33       71d10e102a30   23 hours ago     191
 sh-5.0$ ls -l output/
 total 4
 -rw-r--r--. 1 build build 288 Jul  8 07:48 update-output.txt
-````
\ No newline at end of file
+````
```
Comment (TomSweeneyRedHat) [inline docs/tutorials/05-openshift-rootless-bud.md:418]: I'm not seeing a change here, regardless I think it should be 3 backticks not 4

**Review by saschagrunert — APPROVED** [review]
LGTM

File: docs/tutorials/05-openshift-rootless-bud.md (hunk 2 above)
Comment (nalind) [reply docs/tutorials/05-openshift-rootless-bud.md:418]: Looks like the previous version of the file didn't end with an end-of-line character.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
@TomSweeneyRedHat borrows @rhatdan planet saving hammer.

**Pull request merged**